### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by Jeep2333**
Add the encode api and decode api to the initial code

### CODE REVIEW

File: nosecurity/src/main/resources/static/logincss.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/* Bordered form */
```
Comment (LeonYanc) [inline nosecurity/src/main/resources/static/logincss.css]: Why add this file? As discussed last week, the backend should remove all UI related code html, css, javascript etc.

File: nosecurity/src/main/resources/templates/login-success.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (LeonYanc) [inline nosecurity/src/main/resources/templates/login-success.html]: Same as above

File: nosecurity/src/main/resources/templates/login.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (LeonYanc) [inline nosecurity/src/main/resources/templates/login.html]: Same as above

File: nosecurity/src/main/resources/templates/main.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (LeonYanc) [inline nosecurity/src/main/resources/templates/main.html]: Same a above

**Comment by LeonYanc** [comment]
Please add test evidence, e.g. screen shot to show that the code is running fine and the features like API call and result

**Comment by Jeep2333** [comment]
I just got my charger. The code is written in the previous demo file that's the reason why there is Code related to the UI. I will submit the file one more time after backing into Canada.

Jiejun
________________________________

**Comment by LeonYanc** [comment]
Sure, let's talk about it in the meeting this evening.

Thank you,
Leon

**Comment by Jeep2333** [comment]
Sorry but I believe I did mention that I will not be able to attend this week’s meeting since I am currently in Singapore.

Jiejun
________________________________

**Comment by LeonYanc** [comment]
Last week I saw your email on cancel the meeting. Today I don’t see any email… That’s fine, let’s connect next week. Just please continue your work. The PRs still contain the old code and I have put comments respectively. Please follow the requirements in the separate email and update your PRs.Also a new intern will join us next week.Enjoy your holiday!Thank you,LeonOn Jan 6, 2023, at 9:05 PM, Jeep2333 ***@***.***> wrote:﻿

Sorry but I believe I did mention that I will not be able to attend this week’s meeting since I am currently in Singapore.

Jiejun
________________________________

**Comment by Jeep2333** [comment]
I just uploaded the file one more time with a test screenshot and remove all the UI files.

**Comment by LeonYanc** [comment]
Great, I will check it later.Thank you,LeonOn Jan 12, 2023, at 2:41 PM, Jeep2333 ***@***.***> wrote:﻿
I just uploaded the file one more time with a test screenshot and remove all the UI files.

**Comment by Jeep2333** [comment]
I try to solve the commit problem by git revert but it's not working. I may do it wrong, we probably need to discuss about this during next meeting.
________________________________

**Comment by LeonYanc** [comment]
LGTM

**Review by LeonYanc — APPROVED** [review]
LGTM

**Pull request merged**